src/app/shared/sync.service.ts

```diff
@@ -137,7 +137,9 @@ export class SyncService {
   }
 
   replicatorsArrayWithTags(items, type: 'pull' | 'push', planetField: 'local' | 'parent') {
-    return this.stateService.getCouchState('tags', planetField).pipe(map(tags => this.createReplicatorsArray(items, type, tags)));
+    return this.stateService.getCouchState('tags', planetField).pipe(
+      take(1),
```

Comment:
currently taking only the first observable in the replicators array.

Could compare the 2 received and then take it only if they are the same.